packages/platform/platform-response-filter/src/services/PlatformResponseFilter.spec.ts

```
@@ -328,9 +323,62 @@ describe("PlatformResponseFilter", () => {
+        test() {}
+      }
+
+      const platformResponseFilter = PlatformTest.get<PlatformResponseFilter>(PlatformResponseFilter);
+      const ctx = PlatformTest.createRequestContext();
+      ctx.endpoint = EndpointMetadata.get(Test, "test");
+
+      vi.spyOn(ctx.response, "render").mockRejectedValue(new Error("parser error"));
+
+      ctx.data = {data: "data"};
+
+      let actualError: any = await catchAsyncError(() => platformResponseFilter.transform(ctx.data, ctx));
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'actualError' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)